### DISCUSSION TITLE
Do I need htmx.org dependency?

### DISCUSSION THREAD

**Discussion by LiveNathan** [wimdeblauwe/htmx-spring-boot#59] [Q&A] [answered]
If I have the htmx-spring-boot-thymeleaf dependency, do I also need the htmx.org dependency?

**Comment by wimdeblauwe** [comment, marked as answer]
You do indeed. The easiest way to get started with a project is using the ttcli tool to generate a Spring Boot project and select the "htmx" dependency during the creation. That way, everything is set up correctly for you. See https://github.com/wimdeblauwe/ttcli
